data/campaigns/Heir_To_The_Throne/scenarios/13_The_Dwarven_Doors.cfg

```diff
@@ -236,13 +236,17 @@
         # If you have outlaws in your party, you have the chance to
         # find a special unit
         #
-        [role]
-            type=Footpad,Thug,Poacher,Outlaw,Trapper,Bandit,Fugitive,Ranger,Huntsman,Highwayman
-            role=Outlaw_Advisor
-        [/role]
         [recall]
-            role=Outlaw_Advisor
+            id=Delurin
         [/recall]
+        [role] # If Delurin was just recalled, this will select him.
```

Comment:
I noticed something like this in the Errors PR too, but... how do you know that it'll select Delurin if he was just recalled?